src/v3/src/components/WidgetMessageContainer/WidgetMessageContainer.tsx

```diff
@@ -46,6 +46,46 @@ const WidgetMessageContainer: FunctionComponent<{
     </Typography>
   ));
 
+  const getLinkElement = (link: WidgetMessageLink) => {
+    if (link.isLinkButton) {
+      return (
+        // @ts-expect-error error due to variant type applied, can be ignored
+        <MuiLink
+          component="button"
+          role="link"
+          onClick={(event: ClickEvent) => {
+            event.preventDefault();
+            link.onClick();
+          }}
+          // @ts-expect-error MUI variant type does not include monochrome but functions appropriately when set
```

Comment:
Can you add type augmentations to fix one or either of these TS errors?

Reply:
Oh, I see this was the way it was 😞 Not blocking, but something to consider if you're able to